### ISSUE TITLE
Problem with translations with Get-PnPSiteTemplate and Invoke-PnPSiteTemplate

### ISSUE THREAD

**Issue by tu-developer** [pnp/pnpframework#656]
**Expected behaviour**
In a multilingual site, when using Get-PnPSiteTemplate and Invoke-PnPSiteTemplate all translatable elements must be exported and imported using resx resources.

**Actual behaviour**
Navigation nodes and listview titles are **only** taken into account with the site default language.

**Steps to reproduce behaviour**

(Using a user with owner permissions and default language set to Spanish)

1. Create a communication site with default language set to Spanish
2. Ensure than English is enabled in site languages
3. Create a navigation node and set its title un Spanish and English
4. Create a list and set its default view title in Spanish and English
5. Connect to the site with PnP.PowerShell (1.10) using the same user that creates the site
6. Execute Get-PnPSiteTemplate -Out site-template.pnp
7. Extract site-template.pnp file content to a folder and check the content of the en-US resx file to view that the expected values for the navigation node and the list view title are in English and see that they are in Spanish.
8. Set the correct values in the en-US resx file, and compress the folder in a new site-template.pnp file
9. Create a **new** communication site with default language set to Spanish (with same user)
10. Connect to the new site with PnP.PowerShell (1.10) using the same user that creates the site
11. Execute Invoke-PnPSiteTemplate with the modified site-template.pnp
12. Check the result and see that the corrected text values are not applied and only texts in Spanish are applied.

**Which operating system/environment are you running PnP PowerShell on?**
Windows 11
SharePoint Online
PnP.PowerShell 1.10

**Comment by jagopauwels** [comment]
@tu-developer Did you succeed in fixing this issue?

I'm running into the same issue using PnP Powershell 2.5.0; I tried downgrading to 2.4.0 and 2.3.0 but the bug persists.
We also have a C# project using PnP.Framework version 1.13.0, here we see the same behavior.

Included my test files to reproduce this issue (added .txt to be able to upload)
We apply this template to a Teams site without M365 group.
[NAV.en-US.resx.txt](https://github.com/user-attachments/files/16056048/NAV.en-US.resx.txt)
[NAV.nl-NL.resx.txt](https://github.com/user-attachments/files/16056049/NAV.nl-NL.resx.txt)
[nav.xml.txt](https://github.com/user-attachments/files/16056050/nav.xml.txt)

**Comment by tu-developer** [comment]
> @tu-developer Did you succeed in fixing this issue?
> 
> I'm running into the same issue using PnP Powershell 2.5.0; I tried downgrading to 2.4.0 and 2.3.0 but the bug persists. We also have a C# project using PnP.Framework version 1.13.0, here we see the same behavior.
> 
> Included my test files to reproduce this issue (added .txt to be able to upload) We apply this template to a Teams site without M365 group. [NAV.en-US.resx.txt](https://github.com/user-attachments/files/16056048/NAV.en-US.resx.txt) [NAV.nl-NL.resx.txt](https://github.com/user-attachments/files/16056049/NAV.nl-NL.resx.txt) [nav.xml.txt](https://github.com/user-attachments/files/16056050/nav.xml.txt)

No, the problem was not solved. Back in the day, I tried to look for an API to store/update the texts to make a script to automate it, but I didn't find it. What I finally did was adjust the translations manually after the implementation of the template. In my case it was viable because the number of options was low. It's been a long time, and that API may already exist, but I haven't had to touch anything related to M365 in a long time and I'm not aware of this.
